src/pages/events/troubleshooting.md

```diff
@@ -26,11 +26,15 @@ You can use the following SQL query to select all events from the `event_data` t
 
 `SELECT * FROM event_data`
 
-Use the query results to determine the next troubleshooting step.
+Alternatively, the _Event Status_ grid, found by navigating to **System** > Events > **Event Status** in the Commerce admin, provides a way to view the status of triggered events contained in the `event_data` table through the UI.
```

Comment:
I think we have `Events Status` in the admin menu

Reply:
Yes, you're right. Updated